.github/workflows/stale.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         stale-pr-label: 'status: stale'
         days-before-stale: 90
         days-before-close: 30
-    - uses: actions/stale@v2.0.0
+    - uses: actions/stale@v9.0.0
```

Comment:
Can we use v9 here to get any point releases? (i.e. without the .0.0?)

Reply:
yes, we can. let me change it real quick...

Reply:
Alright, done.